src/main/scala/fundamentals/level02/ListExercises.scala

```diff
@@ -166,4 +166,50 @@ object ListExercises {
     */
   def youngestPerson(persons: List[Person]): Person = ???
 
+  private[level02] val peopleList =
+    List(Person("Matt Murdock", 30),
+         Person("Karen Page", 27),
+         Person("Franklin 'Foggy' Nelson", 31),
+         Person("Claire Temple", 32),
+         Person("Wilson Fisk", 42),
+         Person("Elektra Natchios", 29)
+     )
+
+  /**
+    * Return a list of pairs of a Person and their position in the `peopleList`.
+    * The position should be a 1-based index.
+    *
+    * Hint: Use `zipWithIndex`
+    */
+  def personWithIndex(people: List[Person]): List[(Person, Int)] = ???
+
+  /**
+    * Log every nth person from the `peopleList` given an index `n`.
+    *
+    * scala> showEveryNthPerson(2, peopleList)
+    * = List("Karen Page is 27 years old", "Claire Temple is 32 years old")
+    *
```

Comment:
I think the output would be `List("Karen Page is 27 years old", "Claire Temple is 32 years old","Elektra Natchios is 29 years old")`

Reply:
oh yeah, I added a 6th person and forgot to update the list.